src/components/commanderOverview/CommanderOverview.tsx

```diff
@@ -55,6 +62,13 @@ export const CommanderOverview = React.memo(function MatchHistory() {
                         </Checkbox>
                     </div>
                 </Tooltip>
+                <div style={{ padding: 20 }}>
+                    <Input 
+                        icon='search' 
+                        placeholder='Search...'
+                        onChange={(e) => onSearchChange(e.target.value)}

```

Comment:
can we instantiate this in a useCallback above so we can reuse this instead of creating a new one of these on every render?

Reply:
Done